src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -53,6 +54,16 @@ public partial class WorkspaceViewModel : ViewModelBase
         public bool IsSnapping { get; set; }
 
         /// <summary>
+        /// Gets the Camera Data. This is used when serializing Camera Data in the View block
+        /// of Graph.Json.
+        /// </summary>
+        
+        public CameraData Camera
+        {
+            get { return DynamoViewModel.BackgroundPreviewViewModel.GetCameraInformation(); }
```

Comment:
yep, I just mean, do you think this will contain all the info we require on reach.